src/main/java/de/gessnerfl/fakesmtp/config/SchedulingConfig.java

```diff
@@ -8,11 +9,14 @@
 @Configuration
 public class SchedulingConfig implements SchedulingConfigurer {
 
+    @Value("${fakesmtp.taskScheduler.poolSize:10}")
+    Integer poolSize;
```

Comment:
Instead of just using `@Value` I would move the configuration into the configuration class `FakeSmtpConfigurationProperties`. In the class we could then add an additional subclass for configuring the task Scheduler.